src/main/java/frc/trigon/robot/subsystems/ledstrip/LEDStrip.java

```diff
@@ -0,0 +1,154 @@
+package frc.trigon.robot.subsystems.ledstrip;
+
+import com.ctre.phoenix.led.*;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class LEDStrip extends SubsystemBase {
+    private static final CANdle CANDLE = LEDStripConstants.CANDLE;
+    private static int LAST_CREATED_LED_STRIP_ANIMATION_SLOT = 0;
+    private final int animationSlot;
+    private final int offset, numberOfLEDs;
+
+    /**
+     * @param offset       The offset of how many LEDs you want the LED strip to start from
+     * @param numberOfLEDs The number of LEDs in the strip
+     */
```

Comment:
Our javadoc conventions are the following: 
1. Every javadoc that doesn't only have `@return` should have an explanation for it. Since this is a constructor, the explanation can just be something like "Constructs a new LEDStrip."
2. The explanation should start with a capitalized letter and have a period at the end
3. Every explanation for something that has a `@` before it (i.e. `@return`, `@param`) should start with an uncapitalized letter and shouldn't have a period at the end of it.